tokio/src/runtime/signal/mod.rs

```diff
@@ -24,17 +24,14 @@ pub(crate) struct Driver {
     receiver: UnixStream,
 
     /// Shared state
-    inner: Arc<Inner>,
+    inner: Arc<()>,
 }
 
-#[derive(Clone, Debug, Default)]
+#[derive(Debug, Default)]
 pub(crate) struct Handle {
-    inner: Weak<Inner>,
+    inner: Weak<()>,
```

Comment:
Might be a good idea to document why the Weak is here, because this is going to confuse people.

Reply:
Done.